### ISSUE TITLE
Critical messages when editing columns in hl_tree.f90 and hl_list_n.f90, h_list_renderers.f90 examples (GTK 3.98.4)

### ISSUE THREAD

**Issue by vmagnin** [vmagnin/gtk-fortran#194]
**Describe the bug**
Critical messages when editing columns in hl_tree.f90 and hl_list_n.f90, h_list_renderers.f90 examples:
```
(a.out:2187): Gtk-CRITICAL **: 09:43:02.643: gtk_widget_get_native: assertion 'GTK_IS_WIDGET (widget)' failed
(a.out:2187): Gtk-CRITICAL **: 09:43:02.643: gtk_native_get_surface_transform: assertion 'GTK_IS_NATIVE (self)' failed
(a.out:2187): Gtk-CRITICAL **: 09:43:02.643: gtk_widget_compute_point: assertion 'GTK_IS_WIDGET (widget)' failed
```
**To Reproduce**
```bash
$ gfortran ../src/gtk-hl-container.f90 ../src/gtk-hl-misc.f90 ../src/gtk-hl-button.f90 ../src/gtk-hl-tree.f90 hl_tree.f90 -Wall -Wextra -std=f2003 -pedantic $(pkg-config --cflags --libs gtk-4-fortran --with-path=/usr/local/lib64/pkgconfig/) && ./a.out
```
Edit "Name" or "3N" column.

**Your system:**
 - OS version: Fedora 32 64 bits
 - Compiler version: gfortran 10.1.1
- GTK 3.98.4

**Additional context**
No problem with the GTK 3 version.
These three functions are not called anywhere in our code.

**Comment by vmagnin** [comment]
With GTK 3.98.5, those columns are not editable... But it should work again with GTK 3.99.0:
https://gitlab.gnome.org/GNOME/gtk/-/blob/master/NEWS
> * GtkTreeView:
>     - Make cell editing work again

**Comment by vmagnin** [comment]
The problem was fixed in GTK 3.99.0.
